controller.js

```diff
@@ -120,56 +119,16 @@ Docs:
   }
 
   async uploadAssetsIfNeeded({ buffer, hash }) {
-    if (this.happoDebug) {
-      console.log(`[HAPPO] Checking if we need to upload assets`);
-    }
-
-    try {
-      // Check if the assets already exist. If so, we don't have to upload them.
-      const assetsDataRes = await makeRequest(
-        {
-          url: `${this.happoConfig.endpoint}/api/snap-requests/assets-data/${hash}`,
-          method: 'GET',
-          json: true,
-        },
-        { ...this.happoConfig },
-      );
-      if (this.happoDebug) {
-        console.log(
-          `[HAPPO] Reusing existing assets at ${assetsDataRes.path} (previously uploaded on ${assetsDataRes.uploadedAt})`,
-        );
-      }
```

Comment:
One downside with this change is that we won't be dropping these logs as nicely as we currently do when `this.happoDebug` is truthy.

We should maybe look at how this works across packages and attempt to unify it a bit so it would just work.